packages/notifier/src/subscriber.js

```diff
@@ -35,26 +34,12 @@ harden(makeSubscription);
 export { makeSubscription };
 
 /**
- * @template T
- * @param {ERef<PublicationRecord<T>>} tailP
- * @returns {SubscriptionIterator<T>}
- */
-const makeSubscriptionIterator = tailP => {
-  // To understand the implementation, start with
-  // https://web.archive.org/web/20160404122250/http://wiki.ecmascript.org/doku.php?id=strawman:concurrency#infinite_queue
-  return Far('SubscriptionIterator', {
-    subscribe: () => makeSubscription(tailP),
-    [Symbol.asyncIterator]: () => makeSubscriptionIterator(tailP),
-    next: () => {
-      const resultP = E.get(tailP).head;
-      tailP = E.get(tailP).tail;
-      Promise.resolve(tailP).catch(() => {}); // suppress unhandled rejection error
-      return resultP;
-    },
-  });
-};
-
-/**
+ * @deprecated Producers should use `const { publisher, subscriber } =
+ * makePublishKit(); const topic = makePinnedHistoryTopic(subscriber);`
```

Comment:
```suggestion
 * @deprecated Producers should use
 * ```js
 * const { publisher, subscriber } = makePublishKit(); 
 * const topic = makePinnedHistoryTopic(subscriber);
 * ```
```